### PR TITLE
pkp/pkp-lib#723 Replace citationFormat plugins with CSL integration

### PR DESCRIPTION
**Pull request by NateWr**

### CODE REVIEW

File: js/controllers/list/SelectListPanelItem.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<template>
+	<li class="pkpListPanelItem pkpListPanelItem--select" :class="{'--hasFocus': isFocused}">
```
Comment (asmecher) [inline js/controllers/list/SelectListPanelItem.vue]: Sorry, I haven't dug into this stuff at all yet, but didn't we establish that starting classes off with a double-dash is a problem for Safari?

File: tools/runAllTests.sh [hunk 2]
```diff
@@ -121,7 +121,19 @@ if [ \( "$DO_ALL" -eq 1 \) -o \( "$DO_APP_PLUGINS" -eq 1 \) ]; then
 	if [ \( "$DO_COVERAGE" -eq 1 \) ]; then
 		REPORT_SWITCH="--coverage-php $REPORT_TMP/coverage-APP_PLUGINS.php"
 	fi
-	$phpunit $DEBUG $TEST_CONF2 -v $REPORT_SWITCH plugins
+	$phpunit $DEBUG $TEST_CONF2 -v $REPORT_SWITCH plugins/generic/lensGalley/tests
```
Comment (asmecher) [inline tools/runAllTests.sh]: I worry that listing these out manually will lead to forgotten entries and thus forgotten tests. If you can't specify a wildcard directly to `phpunit`, then maybe a shell script loop through available directories, or a `find -exec` call, would be better.

Comment (NateWr) [reply tools/runAllTests.sh]: I've never written a bash script or set up PHPUnit before. Can you recommend a solution?

File: js/controllers/list/SelectListPanelItem.vue (hunk 1 above)
Comment (NateWr) [reply js/controllers/list/SelectListPanelItem.vue]: You're right, I need to rebase against `master` with these changes in place.

Comment (NateWr) [reply js/controllers/list/SelectListPanelItem.vue]: 6cd4b3d0dca1cfc02b96175cdc40eef40f9df05b

Comment (NateWr) [reply js/controllers/list/SelectListPanelItem.vue]: 01df62e7bde1c79dc91992d9a1f28d75b042cc92

File: tools/runAllTests.sh (hunk 2 above)
Comment (asmecher) [reply tools/runAllTests.sh]: This line should do it:
`find plugins -name tests -maxdepth 3 -type d -exec $phpunit $DEBUG $TEST_CONF2 -v $REPORT_SWITCH "{}" ";"`

Comment (NateWr) [reply tools/runAllTests.sh]: cheers! :+1:

**Pull request merged**